packages/discord.js/src/util/Events.js

```diff
@@ -18,6 +18,7 @@
  * @property {string} GuildAuditLogEntryCreate guildAuditLogEntryCreate
  * @property {string} GuildBanAdd guildBanAdd
  * @property {string} GuildBanRemove guildBanRemove
+ * @property {string} GuildAvailable guildAvailable
```

Comment:
Pretty sure the JSDoc is in alphabetical order.

Reply:
Good catch, resolved in 9f9356d.